app/assets/javascripts/surveys.js

```diff
@@ -0,0 +1,140 @@
+(function() {
+  "use strict";
+
+  var root = this,
+      $ = root.jQuery;
+  if(typeof root.GOVUK === 'undefined') { root.GOVUK = {}; }
+
+  var TEMPLATE = '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
+                 '  <div class="wrapper">' +
+                 '    <h1>Tell us what you think of GOV.UK</h1>' +
+                 '    <p class="right"><a href="#survey-no-thanks" id="survey-no-thanks">No thanks</a></p>' +
+                 '    <p><a href="javascript:void()" id="take-survey" target="_blank">Take the 3 minute survey</a> This will open a short survey on another website</p>' +
+                 '  </div>' +
+                 '</section>';
+
+  var userSurveys = {
+    defaultSurvey: {
+      url: 'https://www.surveymonkey.com/s/2MRDLTW',
+      identifier: 'user_satisfaction_survey',
+      template: TEMPLATE,
+      frequency: 50,
+    },
+    smallSurveys: [],
+
+    init: function() {
+      var activeSurvey = userSurveys.getActiveSurvey(userSurveys.defaultSurvey, userSurveys.smallSurveys);
+      if (userSurveys.isSurveyToBeDisplayed(activeSurvey)) {
+        userSurveys.displaySurvey(activeSurvey);
+      }
+    },
+
+    getActiveSurvey: function(defaultSurvey, smallSurveys) {
+      var activeSurvey = defaultSurvey;
+
+      $.each(smallSurveys, function(_index, survey) {
+        if(userSurveys.currentTime() >= survey.startTime && userSurveys.currentTime() <= survey.endTime) {
+          activeSurvey = survey;
```

Comment:
We identified this limitation during development; a requirement to have multiple surveys "active" at once exists but not until test #7 (We're currently on test #4) and we have a plan for how to address it when we get there.

For the moment, the desired user need is to have the test survey "supersede" the default survey.